### DISCUSSION TITLE
WW3 ERROR IN W3IOGR

### DISCUSSION THREAD

**Discussion by florlopez26** [NOAA-EMC/WW3#664] [Q&A]
Hello  everyone,
I am trying to run the model (latest version) with 5 different grids, and before I get to the ww3_multi program, I have the following errors:

                 *** WAVEWATCH III  Input pre-processing ***
               ===============================================

  Comment character is '$'

 *** WAVEWATCH III ERROR IN W3IOGR :
     PREMATURE END OF mod_def.w FILE

                 *** WAVEWATCH III  Input pre-processing ***
               ===============================================

  Comment character is '$'

 *** WAVEWATCH III ERROR IN W3IOGR :
     PREMATURE END OF mod_def.w FILE

We've been dealing with this error for weeks without being able to solve it, any suggestion on this?

Regards

**Comment by aliabdolali** [comment]
Hi @florlopez26 
do you have 5 mod_def.* files for all of your 5 grids? plus if you have input and point grids, you need mod_def for them as well.

**Reply by florlopez26** [reply]
Hi @aliabdolali 
Thanks for the answer. Yes, I have 5 mod_def.* files and an extra mod_def.* file for the point grid.

**Reply by aliabdolali** [reply]
@florlopez26 but I see `PREMATURE END OF mod_def.w FILE` error which means your mod_defs for at least one of grids is not ok. You should check the standard outputs of ww3_grid and solve it.
